packages/components/src/templates/expandable.stories.ts

```diff
@@ -0,0 +1,110 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Expandable',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/owNwnmTLTH7rDD4kOyGJK9/Expandable?node-id=0-1&t=8M0kKNRUhCykqrGj-0'
+    }
+  }
+};
+
+/**
+ * - `white`
+ * - `neutral-100`
+ * - `primary-100`
+ * - `primary (if inverted)`
+ */
+
+export const BackgroundOptions = {
+  name: 'Background options',
+  render: () =>
+    html`<div class="flex flex-col gap-4">
+      <div class="background-sample">
+        <sd-expandable>
+          <div class="sd-paragraph h-full">
+            For us, diversity equals strength. By taking the views and experiences of a wide range of people into
```

Comment:
For us, diversity equals strength. By taking the views and experiences of a wide range of people into account, we provide room for creative solutions and ensure that we are equipped for the future. A diverse and inclusive working environment is important to us. We benefit from the perspectives of different genders, age groups, educational levels and backgrounds, thus guaranteeing that we are successful together.

As a member of the cooperative financial network, diversity and corporate citizenship are an integral part of our corporate culture. Our approach is based around interacting with each other, our customers and our partners on an equal footing. Our actions are defined by values such as respect, professionalism and collaboration. In the true spirit of the cooperative principles, we channel a wide range of strengths in order to be stronger together.

By signing the Diversity Charter in 2013, Union Investment underlined how highly it values diversity within the company. We want to attract and retain the best candidates, irrespective of cultural or social background, ethnicity, gender, sexual orientation, disability, religion or age. To do this, we have created a working environment that is free of prejudice, where respect is paramount and people can achieve their full potential. This is the policy that underpins the actions of our managers and employees.